src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -49,20 +40,57 @@ public void auxiliaryDrive(double auxiliarySpeed) {
    * @param mainSpeed The speed at which to drive the motor; in [-1, 1].
    */
   public void mainDrive(double mainSpeed) {
-    mainMotor.set(TalonSRXControlMode.Velocity, mainSpeed);
+    io.mainDrive(mainSpeed);
   }
 
   /**
    * Stops the main motor.
    */
   public void stopMain() {
-    mainMotor.set(TalonSRXControlMode.Disabled, 0);
+    io.stopMain();
   }
 
   /**
    * Stops the auxiliary motor.
    */
   public void stopAuxiliary() {
-    auxiliaryMotor.set(TalonSRXControlMode.Disabled, 0);
+    io.stopAuxiliary();
+  }
+
+   /**
+   * Get the voltage applied to the main motor.
+   * @return The applied voltage (in volts).
+   */
+  public double getMainAppliedVolts() {
+    return io.getMainAppliedVolts();
+  }
+
+  /**
+   * Get the current output from the main motor controller.
+   * @return The output/stator current (in amps).
+   */
+  public double getMainCurrentAmps() {
+    return io.getMainCurrentAmps();
```

Comment:
this works -- it's a bit simpler to return inputs.mainCurrentAmps here (and in the other getters)